### PR TITLE
Add of sounds

### PR DESCRIPTION
**Pull request by decommandpro**

### CODE REVIEW

**Comment by TheoPannetier** [comment]
Hey @decommandpro does the WIP here means this isn't ready to be merged? 

if (TRUE) here's a pro tip: you can convert the PR to a draft, which will prevent anyone from merging and closing the PR + tell clearly that you're still working on it 😇 
else
You can request 1-3 reviewers to have a look and merge your branch. Otherwise I fear you may wait for a while

File: game_resources.h [hunk 1]
```diff
@@ -123,6 +137,18 @@ class game_resources
   /// 'bump' sound from Sebastian
   sf::Music m_bump;
 #endif // IS_ON_TRAVIS
+
+#ifndef IS_ON_TRAVIS
+  // Playing sound on Travis gives thousands of error lines, which causes the
+  // build to fail
+  sf::Music m_grow;
+#endif // IS_ON_TRAVIS
+
```
Comment (Yangshen0325) [inline game_resources.h:146]: Line 145 and 147 could be removed?

**Review by Yangshen0325 — COMMENTED** [review]
Cool work!

File: game_resources.h (hunk 1 above)
Comment (swom) [reply game_resources.h:146]: I think it is good to keep `m_grow `and `m_shrink `into 2 separate pre-processor statements blocks

File: sound_type.cpp [hunk 2]
```diff
@@ -1,16 +1,41 @@
...
+  }
+  if (this_sound_type == sound_type::shoot)
+  {
+    return "shoot";
+  }
+  if (this_sound_type == sound_type::hit)
+  {
+    return "hit";
+  }
+  return {};
```
Comment (swom) [inline sound_type.cpp:22]:
```suggestion
throw std::invalid_argument{"sound type not recognized."}
  return {};
```

File: sound_type.cpp [hunk 3]
```diff
@@ -1,16 +1,41 @@
...
+}
+
```
[the remark this answers is not in the document]
Comment (swom) [reply sound_type.cpp:24]: see suggestion above.

**Comment by Yangshen0325** [comment]
Successfully merged.

**Pull request merged**